src/handlers/interventions-processor-handler.ts

```diff
@@ -118,12 +159,17 @@ function isTimestampInFuture(recordBody: TxMAEvent): boolean {
  *
  * @param recordBody - the parsed body of the sqs record
  */
-function getInterventionName(recordBody: TxMAEvent): EventsEnum {
-  logger.debug('Event is valid, starting processing.');
+function getInterventionName(recordBody: TxMAIngressEvent): EventsEnum {
+  logger.debug('event is valid, starting processing');
   if (recordBody.event_name === TICF_ACCOUNT_INTERVENTION) {
     validateInterventionEvent(recordBody);
     const interventionCode = Number.parseInt(recordBody.extensions!.intervention!.intervention_code);
```

Comment:
I have a small question here, what happens if this `recordBody.extensions!.intervention!.intervention_code` object has either extensions or intervention undefined or as null, is that possible to occur in this variable? or does it bubble up and get caught in that try catch?

Reply:
This is not possible to occur because the line above this one validates that the object as those fields. Obviously because this is abstracted out to a separate method (`validateInterventionEvent`) the compiler does not know on line 166 that those fields will for sure exist if that line is being executed, thus why we use non-null assertions. If those fields are undefined or null an error would be thrown on line 165 and it is caught in the caller method

Reply:
Ah I see, thank you!